server/plugins/LFUGens.cpp

```diff
@@ -897,7 +897,9 @@ void LFGauss_Ctor(LFGauss* unit)
 	}
 	unit->mPhase = -1.0;
 
-	//LFGauss_next_k(unit, 1);
```

Comment:
I don't remember why I commented it out. Maybe because it seemed unnecessary at the time, but I'm not sure about that.